singlestoredb/connection.py

```diff
@@ -1288,6 +1288,16 @@ def show(self) -> ShowAccessor:
         """Access server properties managed by the SHOW statement."""
         return ShowAccessor(self)
 
+    @property
```

Comment:
You could use `functools.cached_property` instead.and not have to manually create the behavior.